lepton-schlas fails to link with llvm linker

As the maintainer of the `cad/lepton-eda` `FreeBSD` port,
recently I started receiving email messages like these:
[[package - head-amd64-default][cad/lepton-eda] Failed for lepton-eda-1.9.7 in build](https://lists.freebsd.org/pipermail/freebsd-pkg-fallout/Week-of-Mon-20190401/1205958.html)
[[package - head-i386-default][cad/lepton-eda] Failed for lepton-eda-1.9.7 in build](https://lists.freebsd.org/pipermail/freebsd-pkg-fallout/Week-of-Mon-20190401/1204431.html)
i.e. `lepton-eda` fails to build on `FreeBSD 13.0-CURRENT` (current development branch).
Build fails (on the link stage) in `utils/gschlas/`,
all other components builds without errors.
On `FreeBSD` 11 and 12 there are no errors (binary packages are already available).
On `13.0-CURRENT` the default linker is changed: it is now `ld.lld`
from the `llvm` distribution. It used to be GNU `ld`.


The last lines of the `make` output:

```
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_get_parse_name
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_parse_name
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_io_error_quark
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_equal
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_get_child
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_get_parent
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_get_path
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_get_type
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_hash
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_load_contents
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_make_directory_with_parents
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_new_for_path
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_query_exists
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_query_file_type
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_file_replace_contents
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_input_stream_get_type
ld: error: ../../liblepton/src/.libs/liblepton.so: undefined reference to g_memory_input_stream_new_from_data
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** Error code 1

Stop.
make[4]: stopped in /wrkdirs/usr/ports/cad/lepton-eda/work/lepton-eda-1.9.7-20181211/utils/gschlas
*** Error code 1
```


The functions listed above in `undefined reference` lines are from
the `libgio` library (a part of `glib`).

--- a/utils/gschlas/Makefile.am
+++ b/utils/gschlas/Makefile.am
@@ -16,7 +16,7 @@ lepton_schlas_CPPFLAGS = -I$(top_srcdir)/liblepton/include -I$(srcdir)/../includ
 	-I$(top_srcdir) -I$(includedir)
 lepton_schlas_CFLAGS = $(GCC_CFLAGS) $(MINGW_CFLAGS) $(GLIB_CFLAGS) \
 	$(GUILE_CFLAGS) $(GDK_PIXBUF_CFLAGS)
-lepton_schlas_LDFLAGS = $(GLIB_LIBS) $(GUILE_LIBS) $(GDK_PIXBUF_LIBS)
+lepton_schlas_LDFLAGS = $(GLIB_LIBS) $(GUILE_LIBS) $(GDK_PIXBUF_LIBS) $(GIO_LIBS)
 lepton_schlas_LDADD = $(top_builddir)/liblepton/src/liblepton.la
 
 MOSTLYCLEANFILES = *.log *.ps core FILE *~

